pkg/resources/oauth_integration_test.go

```diff
@@ -37,10 +38,10 @@ func TestOAuthIntegrationCreate(t *testing.T) {
 		).WillReturnResult(sqlmock.NewResult(1, 1))
 		expectReadOAuthIntegration(mock)
 
-		err := resources.CreateOAuthIntegration(d, &internalprovider.Context{
+		err := resources.CreateOAuthIntegration(context.Background(), d, &internalprovider.Context{
 			Client: sdk.NewClientFromDB(db),
 		})
-		r.NoError(err)
+		r.NotEmpty(err)
```

Comment:
because err change from `error` to `diag.Diagnostics` which is a type of `[]diag.Diagnostic`, that's why NotEmpty. I can change e.g. the name from `err` to `diag`, but I kind of skimmed through those files as they will be removed or greatly changed in the future.

Reply:
Changed names to diags and assertion to `r.Empty`